### DISCUSSION TITLE
學校附近的食物選擇(最遠到後門麥當勞)

### DISCUSSION THREAD

**Discussion by YANRONG0123** [1101-GS4719-C/Python-Community-GS4719-C#1943] [General]
想請問各位一下，
由於中央附近店家流動率逐漸攀升，
大家平常的早、午、晚餐通常都吃些什麼呢?
麻煩這邊的討論範圍盡量最遠在「後門麥當勞」就好。

早、午餐(我這個通常一起吃)

0.全家(不多說)

1.喜樂廚房(鍋燒系列真的不錯吃)

2.宵夜屋(想吃什麼自己夾)

3.惟客美而美(一堆套餐直接選)

4.米豆(平常12.還有很多空位)

5.瑞麟美而美(有時突然會想吃?)

6.漢林大漢堡(也是直接選套餐)

7.松果這學期新開的店(三間都還在嘗鮮期)

晚餐

0.全家(不多說)

1.後門麥當勞(限有優惠券時)

2.漢堡王

3.滿食記(飲料好喝，整體不錯，老闆親切)

4.夠義式(還算便宜大碗)

5.緣杏(會開得比較晚，適合晚上有課的人)

6.傻師傅湯包(滿百送飲料)

7.家香(便宜的麵，滷蛋少數10元的店)

8.阿米玲(湯飯菜續到飽)

9.新福

10.一品鍋

11.白鬍子

12.小木屋鬆餅

以上大概是我這兩個學期以來比較常去的店家，
我覺得每個人都會有自己的看法，
不少人感覺都遠離這裡去吃飯，
不知道大家平常在學校附近(最遠在「後門麥當勞」)都吃些什麼呢?

**Comment by SusuTima** [comment]
美式早午餐屋跟粥王後面的早餐店我也蠻常吃的，價格適當，品質正常👍。

**Comment by YifenLin** [comment]
我有時候打完球很餓會跟朋友去吃後門的寶咖咖，可以吃飽又有飲料機讚讚!有提供湯可以喝，環境也算整潔明亮推推

**Reply by Mitochondrionjessica** [reply]
飽咖咖真的一級讚! 上菜速度偏快，飯能吃到飽，老闆娘親切，菜肉給的算大方，價錢也經濟實惠~

**Comment by willychiuchiu** [comment]
推推宵夜街腸粉！便宜又好吃，而且現在內用還有布丁吃到飽，每次都餐前兩碗餐後兩碗才滿足，cp值真的高。但常常看他都沒什麼人，麻煩大家多多去吃不然怕他很快就活不下去了嗚嗚

**Reply by 110101501** [reply]
之前有吃過他們的芋頭布丁，希望能吃到不同口味的布丁。

**Comment by thomasxshen** [comment]
我最喜歡迷路了！迷路的義大利麵又便宜又大碗，味道還很讚。湯還超級好喝，每次去都要喝爆他的湯，而且那邊的員工很正，有成熟大姊姊的味道！迷路絕對是中央最佳的餐廳，可惜每次去都要等很久QAQ

**Comment by tzurou** [comment]
推宵夜街熊讚啦啦，是我吃過最好吃的鹽水雞，滿80有送麵，菜品選擇我覺得不算少，但我通常不太點肉太窮了，這學期有引進線上點餐系統可以選取餐時間不會等太久!不知道吃甚麼或趕時間都可以吃!

**Comment by 20050201Shawn** [comment]
我超愛樂活寶，個人最喜歡他們的薯條跟手工豬肉堡，但如果是在正餐時間通常會等蠻久的。之前看啾啾鞋影片發現他那時候樂活寶就在了，這家店活超久。

**Comment by 155Julia** [comment]
我在早餐店的經驗相當豐富，尤其推薦後門太極飯糰旁邊那家，價格實惠又好吃。他們的奶綠絕了，濃郁得讓人回味無窮，絕對是中央地區最棒的奶綠之一。另外，提到宵夜街，必須一提喜樂廚房，他們的蛋餅無敵，是我在中央吃過最划算、最美味的。外皮酥脆、內餡軟綿，煎得剛剛好。還有他們的焗烤厚片，夠濃夠厚的起司讓人非常滿足。推薦給大家參考。

**Comment by 110101501** [comment]
喜歡宵夜街的鹽水雞！吃起來清爽無負擔，滿額也是會加贈麵條，CP值蠻高！不過買它們家鹽水雞的學生人數多，需要花點時間等待餐點。
順帶一提，他們家的烤布蕾也很好吃，價格也不貴，推推！

**Comment by YOGA012** [comment]
我只吃午餐跟晚餐，午餐我通常在宿舍自己弄，晚餐的話，如果我比較早下課會選宵夜屋，如果晚一點的話大概就後門幾家店輪著吃，昨天驚喜發現羅塔塔搬到後門酸辣粉旁邊，以後應該會常去吃。

**Comment by jim1743** [comment]
如果是吃早午餐，通常會選擇全家或後門的萊姆斯早餐，全家是因為近，比較省時，而選擇萊姆斯是因為他們的飲料比較大杯，餐點上雖然沒有到很好吃，但便宜又吃得飽；晚餐首選是宵夜屋，比較晚下課才會是寶咖咖或新福。

**Comment by cassy0201** [comment]
大推蔬菜蛋餅!吃得飽又相對健康!，但每次禮拜五中午去人都爆滿，吃不到超殘念

**Comment by 710388** [comment]
麥當勞最好吃了，只是非常遠，而且我還沒腳踏車

**Comment by tshsu04224** [comment]
推薦熊讚跟後門的鹹水雞，調味都很好吃🤤有些閒錢的話可以去街角216吃漢堡🍔，他的花生牛肉起司超好吃，每次必點這個！義大利麵的話還得是迷路，迷路的湯真的很好喝，當然義大利麵也是讚的，推薦給你！

**Comment by YeeDaniel** [comment]
我朋友最近帶我去吃萊姆絲，哇不吃不知道，一吃嚇一跳。這個蘿蔔糕真的沒吃過這麼頂的，味道十足，價格又便宜。還有蛋餅也蠻好吃的~ 真的感謝我朋友大力推薦蘿蔔糕~ 分享給你啦
